src/lib.rs

```diff
@@ -11,7 +10,7 @@ use wasm_bindgen::prelude::*;
 static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
 
 #[wasm_bindgen]
-extern {
+extern "C" {
```

Comment:
Why "C"?

Reply:
I have no idea. This was an automated change made once I enabled the VSCode plugin

Reply:
Removed since we don't need this anymore anyways